pkg/ccl/streamingccl/streamingest/stream_replication_e2e_test.go

```diff
@@ -504,9 +503,6 @@ func TestTenantStreamingPauseResumeIngestion(t *testing.T) {
 	srcTime = c.srcCluster.Server(0).Clock().Now()
 	c.waitUntilHighWatermark(srcTime, jobspb.JobID(ingestionJobID))
 	c.compareResult("SELECT * FROM d.t2")
-	// Confirm this new run resumed from the previous checkpoint.
```

Comment:
I guess stashing the current `StartTime` in the ingestion job progress was partly useful to know the start time for that resumption of the job. I don't know if that's reason enough to keep the field.